src/PhylloConnect.ts

```diff
@@ -64,7 +107,16 @@ const PhylloConnectSDK = {
     )
 
     // this is to solely match web sdk signature
-    return { open: () => phyllo.open() }
+    return {
+      open: () => {
+        validateCallbacks(this.callbacksObj)
```

Comment:
Will this still go to `open` if `validateCallbacks` fails?

Reply:
@akshaybhalotia

If any error occur, the sdk flow will not open, it'll throw the error instead...